src/client/app/directives/directives.js

```diff
@@ -0,0 +1,18 @@
+interview_coding_app.directive('ngConfirmClick', [
+  function(){
+    return {
+      priority: -1,
+      restrict: 'A',
+      link: function(scope, element, attrs){
+        element.bind('click', function(e){
+          var message = attrs.ngConfirmClick;
+          // confirm() requires jQuery
+          if(message && !confirm(message)){
+            e.stopImmediatePropagation();
+            e.preventDefault();
+          }
+        });
+      }
+    }
+  }
```

Comment:
Interesting approach to getting a confirmation from the user before removing the item. Is there another, simpler solution?